Linux環境下において、ソースからの起動・ビルドができない

### Description of the bug / バグの説明

[v0.1.4](https://github.com/vrc-get/vrc-get/tree/gui-v0.1.4) 以降において、Linux環境下で`npm run tauri dev`並びに`cargo tauri dev`による起動ができない

### Steps to Reproduce / 再現手順

1. GitHubよりソースをcloneする
2. vrc-get-guiディレクトリ下にて `npm run tauri dev` を実行
3. 下記エラーにより起動不可

```
error: linking with `cc` failed: exit status: 1
  |

/* ccのコンパイルコマンドのため中略 */

= note: /usr/bin/ld: /tmp/rustcWK0QUV/libbootstrapperdll.a(bootstrapperdll.o): in function `InitializeRuntime()':
          /__w/1/s/src/coreclr/nativeaot/Bootstrap/main.cpp:179:(.text._ZL17InitializeRuntimev+0x5f): undefined reference to `RhRegisterOSModule'
          collect2: error: ld returned 1 exit status
          
  = note: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-link-lib)

warning: `vrc-get-gui` (bin "vrc-get-gui") generated 2 warnings
error: could not compile `vrc-get-gui` (bin "vrc-get-gui") due to 1 previous error; 2 warnings emitted
```


### Expected Behavior / 想定される挙動

本来であれば特にエラー等なくALCOMが立ち上がるはずだが、上記エラーにより立ち上がらない

### Screenshots / スクリーンショット

_No response_

### Software Version / バージョン情報

gui-0.1.4 to gui-0.1.7

### OS

Arch Linux x86_64 kernel 6.9.3-arch1-1

### Additional context / 補足

Linuxディストロ起因の可能性も否定できなかったため、Docker上のDebian 12環境でも検証しましたが、同様のエラーにより起動できませんでした。

--- a/vrc-get-litedb/build.rs
+++ b/vrc-get-litedb/build.rs
@@ -87,8 +87,8 @@ fn main() {
 
     let common_libs: &[&str] = &[
         //"static=Runtime.ServerGC",
-        "static=Runtime.WorkstationGC",
-        "static=eventpipe-disabled",
+        "static:-bundle=Runtime.WorkstationGC",
+        "static:-bundle=eventpipe-disabled",
     ];
 
     for lib in common_libs {

--- a/vrc-get-litedb/build_target_info.rs
+++ b/vrc-get-litedb/build_target_info.rs
@@ -46,7 +46,10 @@ impl TargetInformation {
         Self {
             dotnet_runtime_id: rid,
             output_file_name: "vrc-get-litedb.a",
-            link_libraries: vec!["static=System.Native", "static=stdc++compat"],
+            link_libraries: vec![
+                "static:-bundle=System.Native",
+                "static:-bundle=stdc++compat",
+            ],
             bootstrapper: "libbootstrapperdll.o",
             patch_mach_o: false,
             family: TargetFamily::Linux,

